modules/odc-customizing-available-cluster-roles-using-the-web-console.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-the-web-console_{context}"]
+= Customizing the available cluster roles using the web console
+
+In the *Developer* perspective of the web console, the *Project* -> *Project access* page enables a project administrator to grant roles to users in a project. By default, the available cluster roles that can be granted to users in a project are *admin*, *edit*, and *view*.
```

Comment:
The roles specified here were formatted using underscores rather than asterisks earlier in this PR. Which is most appropriate? Does it make a difference?

Reply:
@kowen-rh I think, in all cases, there should be no markup, as we are talking conceptually about the roles and not as part of the UI. I removed the markup.